src/components/BankTransactions/BankTransactions.tsx

```diff
@@ -151,16 +151,14 @@ const BankTransactionsContent = ({
 
   useEffect(() => {
     if (JSON.stringify(inputFilters) !== JSON.stringify(filters)) {
-      /** @TODO - Do we want to default filters to fetch categorized and uncategorized transactions when categorization is disabled? */
-      /* if (!categorizationEnabled) {
+      if (effectiveBookkeepingStatus === 'ACTIVE') {
```

Comment:
Use `all` (categorized and non-categorized) bank transactions when bookkeeping active